changelog.md

```diff
@@ -1,3 +1,9 @@
+## Changes from 1.3.13 to 1.3.14
+
+- [MARATHON-7974](https://jira.mesosphere.com/browse/MARATHON-7974) Switched AuthorizedAction to Contravariance instead of Covariance, which enforces the correct resource authorizations through compilation.   This fix resulted in a security fix for GroupResources.   The issue is such that a non super user provided correct permissions to update a group would always be denied access.   This is now resolved.
```

Comment:
isn't the contravariance vs covariance too much implementation details? Maybe we should make it more user focused meaning write only what the user might be interested in ... ?

Reply:
I've always struggled with this...   I like facts and expect high tech users... that said... I will take a stab at focused on functional benefit.